source/userguide/auto_annotator.rst

```diff
@@ -0,0 +1,437 @@
+Working with the Auto Annotator
+=====================================================
+
+The Auto Annotator
+
+  - is a tool to automatically annotate or unannotate select entities across all labelled data in an application.
+  - supports the development of custom Annotators.
+
+.. note::
+
+   The examples in this section require the :doc:`HR Assistant <../blueprints/hr_assistant>` blueprint application. To get the app, open a terminal and run ``mindmeld blueprint hr_assistant``.
+
+.. warning::
+
+   Changes by an Auto Annotator cannot be undone and Mindmeld does not backup query data. We recommend using version control software such as Github.
+
```

Comment:
We should add a Quick Start section telling how to use these commands directly:
```
mindmeld annotate --app-path "hr_assistant"
mindmeld unannotate --app-path "hr_assistant"

Alternatively
.. code-block:: python
	from mindmeld.auto_annotator import SpacyAnnotator 
	sa = SpacyAnnotator(app_path="hr_assistant")
	sa.annotate()

Reply:
Added a Quick Start section